### DISCUSSION TITLE
Docker - Open Workspace Folder on Startup

### DISCUSSION THREAD

**Discussion by Les-Linux** [gitpod-io/openvscode-server#470] [Q&A]
Hello,

I built an image of gitpod-io/openvscode-server which includes my dependencies. I mount local directory /tmp into the container which I can access. I am trying to figure out next, how to have vscode open the mounted directory as the default workspace on startup and have tried to pass varied parameters to the Docker Run command, but to no avail - example:
`docker run -it --init -p 3000:3000 --mount "type=bind,src=/tmp,dst=/home/workspace" --env "-- workspace /home/workspace/" openvscode-server `

How do I get openvscode-server to open a default workspace on startup, regardless of browser in use (i.e. without having to do this from within vscode)?

With best regards.

Les

**Comment by Les-Linux** [comment]
What I have got working as a workaround is to call the ULR and including a folder as a URL parameter - 
`http://127.0.0.1:3000?folder=/home/workspace`.

I would also like to understand if this can be done with a Docker RUN statement?

**Comment by jeanp413** [comment]
@Les-Linux there's `--default-folder` and `--default-workspace` options when running the server, and yes it should work passing them to the `docker run` command
